### DISCUSSION TITLE
Launch commands from seperate terminal by class name?

### DISCUSSION THREAD

**Discussion by Momijie** [kovidgoyal/kitty#7146] [Q&A]
I have a scratchpad kitty terminal with the class name 'kitty-scratch' 
I have another separate kitty terminal that's opened through dmenu. 
How do I send a launch request from my dmenu opened kitty terminal to my kitty-scratch terminal?
when I check kitty @ ls on each one, they both have the id value of 1.

**Comment by kovidgoyal** [comment]
You use remote control with sockets. See the --listen-on option in kitty
--help
